### ISSUE TITLE
pre_transform does not work

### ISSUE THREAD

**Issue by guochengqian** [pyg-team/pytorch_geometric#496]
## To Reproduce

1. train_dataset = ModelNet(args.data_path, '10', True, pre_transform=T.NormalizeScale())
2. for i, data in enumerate(train_loader)
3. pos = data.pos

## Expected behavior

value range of pos should be (-1, 1), which is (-1156.0836, 1156.1405)
## Environment

* OS: Ubuntu 16
* Python version: 3.7
* PyTorch version: 1.1
* CUDA/cuDNN version: 9.1

Thank you.

**Comment by rusty1s** [comment]
When using the `pre_transform`, examples will get transformed before saving them to disk. So the `pre_transform` gets only called on dataset creation. I guess you already have a processed ModelNet dataset on your disk, so you need to remove `args.data_path/processed` first.

**Comment by guochengqian** [comment]
Thank you for your quick response. I will check whether it works ASAP. Many thanks.
